pyaedt/modules/solutions.py

```diff
@@ -1088,18 +1091,21 @@ def ifft_to_file(
 
 
 class FfdSolutionData(object):
-    """Contains information from the far field solution data.
+    """Antenna array far-field data class.
 
-    Load far field data from the element pattern files.
+    Read embedded element patterns generated in HFSS and provide the Python interface
+    to plot and analyze the array far-field data.
 
     Parameters
     ----------
     eep_files : list or str
-        List of element pattern files for each frequency.
-        If the input is string, it is assumed to be a single frequency.
+        List of embedded element pattern files for each frequency.
+        If data is only provided for a single frequency, then a string may be passed
```

Comment:
```suggestion
        If data is only provided for a single frequency, then a string can be passed
```
May should be reserved for granting permission. Use "can" or "might" instead.